### PR TITLE
Fixes eventype form

### PR DESCRIPTION
**Pull request by zomars**

### CODE REVIEW

**Review by pumfleet — APPROVED** [review]
Code looks good to me, haven’t tested locally though

File: pages/event-types/[type].tsx [hunk 1]
```diff
@@ -1285,16 +1276,14 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
...
-        },
-      })
-    );
+    const fallbackUser = await prisma.user.findUnique({
+      where: {
+        id: session.user.id,
+      },
+      select: userSelect,
+    });
+    if (!fallbackUser) throw Error("The event type doesn't have user and no fallback user was found");
```
Comment (emrysal) [inline pages/event-types/[type].tsx:1285]: The situation will occur when there is a team event with zero attendees. See above suggestion to resolve.

File: pages/event-types/[type].tsx [hunk 2]
```diff
@@ -1285,16 +1276,14 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
 
   // backwards compat
   if (eventType.users.length === 0) {
```
Comment (emrysal) [inline pages/event-types/[type].tsx:1278]:
```javascript
if (eventType.users.length === 0 && !eventType.team)
```

**Review by emrysal — APPROVED** [review]

File: pages/event-types/[type].tsx (hunk 2 above)
Comment (zomars) [inline pages/event-types/[type].tsx:1278]:
```suggestion
  if (eventType.users.length === 0 && !eventType.team) {
```

File: pages/event-types/[type].tsx (hunk 1 above)
Comment (zomars) [reply pages/event-types/[type].tsx:1285]: Nice catch!

**Pull request merged**